### ISSUE TITLE
Images NOT showing if you use browser Back button to go back to the previous page with yall lazy loaded images

### ISSUE THREAD

**Issue by Badhumvee** [malchata/yall.js#38]
Images NOT showing if you use browser Back button to go back to the previous page with yall lazy loaded images.  That previous page loaded fine initially, but then you click on a link and navigate to second page ... then you hit browser back button (chrome and edge) the images that you saw on the first page are not visible anymore and instead, in developer tools you see the 1px-1px  placeholder.jpg image in place of the real image.  
Maybe we are doing something wrong.  Could you please advise?  Thanks a lot.  

TS

**Comment by malchata** [comment]
Do these images reappear if you scroll or otherwise interact with the page after hitting the back button? In any event, I'll need a minimally reproducible test page (or pages in this case) to see the effect in action.

**Comment by Badhumvee** [comment]
https://docs.google.com/document/d/1gIxTtMXDW4ggoIJ-Ts1y33zlLX-ZeiCVg0EjbQszyFQ/edit?usp=sharing

Once you open that URL page in Chrome, we load a bunch of images using yall with the following settings:
```
       document.addEventListener("DOMContentLoaded", function() {
           yall({
               observeChanges: true,
               idlyLoad: true,
               threshold: 600
           });
       });
```

Then, just click on Sign-up on the top right corner.  
Then click on the browser back button.  At this point, we do not see any of the images we saw before.  

I dont feel like it has anything to do with the idlyLoad issue.  

Thanks for your help.

**Comment by malchata** [comment]
Try dropping the attached version in to see if this helps:
[yall-2.2.1.min.js.zip](https://github.com/malchata/yall.js/files/2729675/yall-2.2.1.min.js.zip)

I need to test this more in other browsers, but when I use local sources in Chrome to test this on a local version of your site, it appears to work. It adds the `subtree` option in the `MutationObserver` option, which is what I _think_ fixes this issue.

**Comment by Badhumvee** [comment]
Hi malchata, Thanks for the updated file.  I tried it locally, and it seems to fix this issue.  I expect it will be fine when deployed.  Will let you know if we see any issues.  Thanks again.

**Comment by malchata** [comment]
Great. I'll get a new release prepared and close this. Thanks for getting
back to me!

**Comment by malchata** [comment]
Officially fixed and released as [version 2.2.1](https://github.com/malchata/yall.js/releases/tag/2.2.1).
